front/backup.php

```diff
@@ -586,7 +585,7 @@ function backupMySql($DB, $dumpFile, $duree, $rowlimit) {
    echo "<br/>" . __('You should rather use a dedicated tool on your server.');
    echo "</li></ul></div>";
    echo "</td></tr><tr><td>";
-   echo "<img src='".$CFG_GLPI["root_doc"]."/pics/sauvegardes.png' alt=\"".__s('Deleted')."\">".
```

Comment:
Is this image still used? It can be removed if not.

Reply:
I had no idea why it is there. It appears to be decorative, but has a tooltip of 'Deleted'. It's also used in the Rules menu but with a tooltip 'Users'. I could remove or replace with Font Awesome save icon.

Reply:
Just remove the `alt` here, there is no alt on `i` tags; and it should have been empty anyways (decorative images must have an empty `alt`)... Do not change anything else , it's out of the scope of the current pull request ;)

Reply:
OK, I'll do the simage change on my side for 9.3/bf and will rebase your PR once done. you do not need to worry about that, thanks